Mage.Sets/src/mage/cards/k/KindredDominance.java

```diff
@@ -55,7 +57,6 @@ public KindredDominance(UUID ownerId, CardSetInfo setInfo) {
         super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{5}{B}{B}");
 
         // Choose a creature type. Destroy all creatures that are not the chosen type.
-        this.getSpellAbility().addEffect(new ChooseCreatureTypeEffect(Outcome.DestroyPermanent));
```

Comment:
Probably we need to update KindredDominanceEffect text if this part is removed.

Reply:
Yes, we do.  Fixed.  Sorry about that..